### DISCUSSION TITLE
Permission Role by Collections

### DISCUSSION THREAD

**Discussion by jorge6242** [directus/directus#16727] [Archive]
Is possible create roles, for only create or update models?, this role only have the permissions only for the content, but not have the permission to saw the data model option

![Screen Shot 2022-12-05 at 5 37 06 PM](https://user-images.githubusercontent.com/15070448/205752783-63adf7d3-4d90-4bc2-bac4-8471ba90306c.png)

![Screen Shot 2022-12-05 at 5 37 43 PM](https://user-images.githubusercontent.com/15070448/205752738-ca4bb045-aa9d-4aae-9b8a-2712e668d2fb.png)

**Comment by azrikahar** [comment]
Currently a role must have "Admin Access" enabled in order to view/edit anything inside the Settings module, which includes the Data Model 👍 Reference: https://docs.directus.io/configuration/users-roles-permissions/roles.html#configure-role-details

Additional quote from the [Settings module](https://docs.directus.io/app/settings.html) docs:

> To maintain project safety and integrity, this module is only available to Administrators.

**Reply by jorge6242** [reply]
Thank you by the answer bro, but will be great if the CMS can have permission by table as well
